### PR TITLE
global: standardize timezone

### PR DESCRIPTION
**Pull request by BadrAly**
* Fixes areas where utc timezone was not set.

Co-Authored-by: Aly Badr aly.badr@rero.ch

### CODE REVIEW

File: rero_ils/modules/items/api.py [hunk 1]
```diff
@@ -563,13 +563,13 @@ def action_filter(self, action, loan):
...
             extension_count = loan.get('extension_count', 0)
             if not (
-                circ_policy.get('number_renewals') > 0 and
-                extension_count < circ_policy.get('number_renewals') and
-                extend_loan_data_is_valid(
-                    loan.get('end_date'),
-                    circ_policy.get('renewal_duration'),
-                    self.library_pid
-                )
+                    circ_policy.get('number_renewals') > 0 and
```
Comment (rerowep) [inline rero_ils/modules/items/api.py]: What is the PEP for intended `if`

File: rero_ils/modules/items/api.py [hunk 2]
```diff
@@ -578,9 +578,9 @@ def action_filter(self, action, loan):
 
         if action == 'receive':
             if (
-                circ_policy.get('allow_checkout') and
-                loan.get('state') == 'ITEM_IN_TRANSIT_FOR_PICKUP' and
-                loan.get('patron_pid') == patron_pid
+                    circ_policy.get('allow_checkout') and
```
Comment (rerowep) [inline rero_ils/modules/items/api.py]: What is the PEP for intended `if`

File: rero_ils/modules/items/api.py [hunk 3]
```diff
@@ -824,7 +823,7 @@ def automatic_checkin(self):
                 LoanAction.NO: None
             }
 
-        elif self.status == ItemStatus.MISSING:
+        if self.status == ItemStatus.MISSING:
```
Comment (rerowep) [inline rero_ils/modules/items/api.py]: If you not using `elif` the `self.status`test will be rerun even the status is `IN_TRANSIT`

File: rero_ils/modules/items/api_views.py [hunk 4]
```diff
@@ -307,9 +304,9 @@ def item(item_barcode):
                 else:
                     new_actions.append(action)
             if (
-                action == 'receive' and
-                circ_policy.get('allow_checkout') and
-                item.number_of_requests() == 0
+                    action == 'receive' and
```
Comment (rerowep) [inline rero_ils/modules/items/api_views.py]: What is the PEP for intended if

File: rero_ils/modules/libraries/api.py [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 """API for manipulating libraries."""
 
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from functools import partial
 
 from dateutil import parser
```
Comment (rerowep) [inline rero_ils/modules/libraries/api.py]: Is the `parser` still used?

File: rero_ils/modules/loans/cli.py [hunk 6]
```diff
@@ -224,26 +224,26 @@ def get_random_pickup_location(patron_pid):
     return random.choice(pickup_locations_pids)
```
Comment (rerowep) [inline rero_ils/modules/loans/cli.py]: Do we have to save the this code?

File: rero_ils/modules/items/api.py (hunk 1 above)
Comment (BadrAly) [reply rero_ils/modules/items/api.py]: As discussed

File: rero_ils/modules/items/api.py (hunk 3 above)
Comment (BadrAly) [reply rero_ils/modules/items/api.py]: fixed

File: rero_ils/modules/libraries/api.py (hunk 5 above)
Comment (BadrAly) [reply rero_ils/modules/libraries/api.py]: yes

File: rero_ils/modules/loans/cli.py (hunk 6 above)
Comment (BadrAly) [reply rero_ils/modules/loans/cli.py]: removed.

File: rero_ils/modules/loans/cli.py [hunk 7]
```diff
@@ -108,7 +108,7 @@ def create_loan(barcode, transaction_type, loanable_items):
     """Create loans transactions."""
     item = next(loanable_items)
     patron = Patron.get_patron_by_barcode(barcode=barcode)
-    start_date, end_date = get_loan_dates(transaction_type, item, patron)
+    # start_date, end_date = get_loan_dates(transaction_type, item, patron)
```
Comment (jma) [inline rero_ils/modules/loans/cli.py]: you can remove this comment

File: rero_ils/modules/notifications/api.py [hunk 8]
```diff
@@ -131,7 +131,7 @@ def replace_pids_and_refs(self):
                 data['loan']['author'] = author
             end_date = data.get('loan').get('end_date')
             if end_date:
-                end_date = ciso8601.parse_datetime_as_naive(end_date)
+                end_date = ciso8601.parse_datetime(end_date)
```
Comment (jma) [inline rero_ils/modules/notifications/api.py]: can you use `date_string_to_utc` instead?

File: tests/api/test_external_services.py [hunk 9]
```diff
@@ -56,7 +56,8 @@ def test_documents_get(client, document):
     assert res.status_code == 200
     data = get_json(res)
     # TODO: Implement another solution for available on holding level
-    del(data['hits']['hits'][0]['metadata']['available'])
+    if data['hits']['hits'][0]['metadata'].get('available'):
```
Comment (jma) [inline tests/api/test_external_services.py]: is it really related to timezone?

File: rero_ils/modules/notifications/api.py (hunk 8 above)
Comment (BadrAly) [reply rero_ils/modules/notifications/api.py]: done

File: tests/api/test_external_services.py (hunk 9 above)
Comment (BadrAly) [reply tests/api/test_external_services.py]: removed.

**Pull request merged**